uv.gyp

```diff
@@ -451,6 +451,7 @@
         'test/test-udp-open.c',
         'test/test-udp-options.c',
         'test/test-udp-send-and-recv.c',
+        'test/test-udp-send-hang-loop.c',
```

Comment:
I believe this has to be added to `Makefile.am` and `checkparse.sh` too. cc @trevnorris